### ISSUE TITLE
Pyperclip could not find a copy/paste mechanism for your system.

### ISSUE THREAD

**Issue by x-jalil** [x-jalil/secretum#1]
Pyperclip cannot copy text to the system clipboard.

```
pyperclip.PyperclipException: 
    Pyperclip could not find a copy/paste mechanism for your system.
```

Solution: Install `xclip` in the system
```
sudo apt install xclip
```

**Comment by x-jalil** [comment]
Commit ba99efa81552109165fb90fd7ba3eff5036d6625 close #1

**Comment by x-jalil** [comment]
Commit ba99efa81552109165fb90fd7ba3eff5036d6625 closes #1
